packages/commonwealth/client/scripts/views/components/chat/chat_window.tsx

```diff
@@ -201,17 +204,18 @@ export class ChatWindow implements m.Component<ChatWindowAttrs> {
               app.socket.chatNs.isConnected ? '' : ' disabled'
             }`}
           >
-            <QuillEditor
+            <QuillEditorComponent
+              // TODO Graham 7/20/22: I hate this usage of contentsDoc—can it be improved?
```

Comment:
What do we use it for?

Reply:
We use it basically to force the editor to be blank, rather than be overwritten by local storage stuff/saved drafts.